Add new snippet (#5)
olivierloverde/minutes-of-javascript: README.md

21. [Using "_" in numbers for better readability](snippets/21-tips-underscore-number.js)
22. [Array: .map() vs .filter() vs .reduce()](snippets/22-map-vs-filter-vs-reduce.js)
23. [Generator function* and yield usage](snippets/23-generator-function-yield-usage.js)

> 24. [Use switch/case statements with numeric ranges](snippets/24-use-switch-case-with-numeric-ranges.js)

Reviewer: ```suggestion
24. [Usage of switch (true), an example with numeric ranges](snippets/24-use-switch-case-with-numeric-ranges.js)
```
